### ISSUE TITLE
Invalid program name

### ISSUE THREAD

**Issue by volodya-lombrozo** [objectionary/lints#338]
I'm getting the following error:
```
Value 'JnaPrinter$CLibrary' is not facet-valid with respect to pattern '[a-z]\S*' for type 'program-name'.;
cvc-attribute.3: The value 'JnaPrinter$CLibrary' of attribute 'name' on element 'program' is not valid with respect to its type, 'program-name'.
```

But `JnaPrinter$CLibrary` is a perfect and legal program name. Can you fix this check, please?

**Comment by volodya-lombrozo** [comment]
@yegor256 Could you take a look, please?

**Comment by volodya-lombrozo** [comment]
blocks https://github.com/objectionary/jeo-maven-plugin/pull/1003

**Comment by volodya-lombrozo** [comment]
@yegor256 I have the following inside `JanPrinter$CLibrary`:

```xml
name="j$JnaPrinter$CLibrary"
```

Does the original message mean the file name?

**Comment by yegor256** [comment]
@volodya-lombrozo the message above is about `attribute 'name' on element 'program'`

**Comment by volodya-lombrozo** [comment]
@yegor256 I have the following `program` definition:

```xml
<program xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         dob="2025-02-13T10:55:48"
         ms="0"
         name="j$JnaPrinter$CLibrary"
         revision="c949d71"
         time="2025-02-13T07:56:09.785553Z"
         version="1.0-SNAPSHOT"
         xsi:noNamespaceSchemaLocation="https://www.eolang.org/xsd/XMIR-0.51.4.xsd">
```

**Comment by yegor256** [comment]
@volodya-lombrozo are you sure you are checking the right document? This messages says that the value of `@name` is different:

```
Value 'JnaPrinter$CLibrary' is not facet-valid with respect to pattern '[a-z]\S*' for type 'program-name'.;

```

**Comment by volodya-lombrozo** [comment]
@yegor256 The original problem is related to `objectionary/eo` repository:
https://github.com/objectionary/lints/issues/338

So, I believe we can close this one.
